instrumentation/logger/lib/opentelemetry/instrumentation/logger/patches/logger.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Logger
+      module Patches
+        # Instrumention for methods from Ruby's Logger class
+        module Logger
+          attr_writer :skip_instrumenting
+
+          def format_message(severity, datetime, progname, msg)
+            formatted_message = super
+            return formatted_message if skip_instrumenting?
+
+            OpenTelemetry.logger_provider.logger(
+              name: OpenTelemetry::Instrumentation::Logger::NAME,
+              version: OpenTelemetry::Instrumentation::Logger::VERSION
+            ).on_emit(
+              severity_text: severity,
+              severity_number: severity_number(severity),
+              timestamp: datetime,
+              body: msg,
```

Comment:
New Relic uses `formatted_message` here. This includes both the progname and the message in the body, whereas `msg` does not include the progname. Which seems more appropriate?